src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -45,7 +46,9 @@
 	});
 
 	$: loading.set(
-		isRouteTransactions($page) ? isNullish(pageTokenUi?.balance) : !$exchangeInitialized
+		isRouteTransactions($page)
+			? isNullish(pageTokenUi?.balance)
+			: !EXCHANGE_DISABLED && !$exchangeInitialized
```

Comment:
If the `EXCHANGE_DISABLED`, should `$exchangeInitialized` always be true?

Reply:
I thought about it and i didn't want to touch that derived, because in the end, it is not initialized, right? it is useful to have a derived that checks the initialization even if the exchange is enabled

Reply:
On the other hand, we use `$exchangeInitialized` in various places, so we are introducing a discrepancy here that is not covered by any particular tests.

Reply:
mmmm probably you are right... in the end, that derived store shall say that the exchange are "initialized" if they are not enabled